test/evergreen/import_compatibility_test.sh

```diff
@@ -145,8 +145,6 @@ build_sys['develop']="cmake"
 build_sys['mongodb-5.0']="autoconf"
 build_sys['mongodb-4.4']="autoconf"
 build_sys['mongodb-4.2']="autoconf"
-build_sys['mongodb-4.0']="autoconf"
```

Comment:
This one too @lukech ?

Reply:
This line does not seem to be used in the script. `release_branches` variable does not include `mongodb-4.0` as live import was supported starting from 4.2. I'd happy this line to be removed as a cleanup.

Reply:
OTOH, it looks this test misses coverage for the latest "mongodb-6.0" branch.

Reply:
@lukech I can add mongodb-6.0 branch to this test, as part of [WT-9254](https://jira.mongodb.org/browse/WT-9254).

Reply:
Awesome, thanks @Siddhartha8899